javascript/updatable/inner_updates_compat.js

```diff
@@ -0,0 +1,37 @@
+export const registerInnerUpdates = () => {
+  document.addEventListener('stimulus-reflex:before', event => {
+    recursiveMarkUpdatesForElements(event.detail.element)
+  })
+
+  document.addEventListener('stimulus-reflex:after', event => {
+    setTimeout(() => {
+      recursiveUnmarkUpdatesForElements(event.detail.element)
+    })
+  })
+
+  document.addEventListener('turbo:submit-start', event => {
```

Comment:
Should we consider adding event listeners for `turbolinks`?

Reply:
My memory may not serve me well today but IIRC turbolinks had no submit-start and submit-end events

Reply:
Here's the list, FWIW: https://github.com/turbolinks/turbolinks#full-list-of-events

There's no `submit-start` or `submit-end`, but it's not clear if it's just been renamed or is an entirely new concept.

Reply:
Well since form handling via turbo is all-new I think probably the latter is the case.

And it really only applies to form submits anyway.

But you raise an interesting point about extension. What if we provided a way to add additional events to this list of listeners? 🤨

Reply:
Agreed